src/test/components/__snapshots__/TrackNetwork.test.js.snap

```diff
@@ -714,7 +714,7 @@ exports[`timeline/TrackNetwork matches the component snapshot 1`] = `
       class="timelineVerticalIndicatorsLine"
       data-marker-index="12"
       data-testid="vertical-indicator-line"
-      style="--vertical-indicator-color: var(--blue-50); left: 200px;"
+      style="--vertical-indicator-color: var(--blue-50); left: 266.6666666666667px;"
```

Comment:
And this one?

Reply:
Same for this one too. Now the duration markers show the correct place.